lib/master.js

```diff
@@ -0,0 +1,62 @@
+var cluster = require('cluster'),
+    noop = require("./noop");
+
+module.exports = exports = function(jasmine, env, print) {
+  var runJasmine = require('./run');  // for spy purpose
+  jasmine.execute = noop;
```

Comment:
Why are we clobbering the `execute` function on the underlying Jasmine object here.

Reply:
master does not need to execute the test because this is the workers' job. If I do not put noop, it will run the test again. Keep that in mind `./run ` is shared with not only the workers but also the master if people disable the parallelism

Reply:
I like the concept of using the master to collect results from workers, but maybe we need a different thing for it to execute when we're in clustered mode so that we don't need to patch over existing functionality.

Reply:
Sorry. I have no idea what the different is because master has nothing to do while the workers are running the tests except collecting and showing the result.

Reply:
I guess I'm wondering whether the master thread needs to do much (anything?) Jasmine execution related if it is really just an orchestrator. Maybe there needs to be a file that just loads configuration that both the orchestrator and the workers can use to make sure they are configured correctly without that file immediately executing the suite. Sorry for the confusion.

Reply:
I guess `lib/run.js` is the one you describe because both orchestrator and worker will call for reading the configuration. maybe `run.js` is a bad name so you do not realize it. I do not know how to name it because `initial` is used in `command.js`

Reply:
Yes, it seems like removing the call to `execute()` in `lib/run.js` and renaming it to something like `loadConfig.js` (for example) would mean that `master.js` doesn't need to overwrite the existing `execute` function on the Jasmine it is being passed, since it just won't call it.

Reply:
Please check